clients/docs/includes/configs/cloud/hocon-sr.config

```diff
@@ -21,6 +21,8 @@ kafka {
     key.deserializer = org.apache.kafka.common.serialization.StringDeserializer
     value.deserializer = io.confluent.kafka.serializers.json.KafkaJsonSchemaDeserializer
     json.value.type = io.confluent.developer.Question
+    # Recommended consumer configuration for higher availability in Apache Kafka clients prior to 3.0
```

Comment:
It might be nice if the comment made it clear that 45s is the new default for newer clients. How about something like this?

```
# Increase session timeout for higher availability (45s is the default for Apache Kafka 3.0 and above)
```